content/aips/AGD-RENEWAL-JAN-2023.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Renew Aave Grants DAO
+status: Proposed
+author: Bill (0xbilll) and Llama
+shortDescription: Provide funding to expand the budget and scope of Aave Grants DAO
+discussions: https://governance.aave.com/t/aave-grants-dao-update-renewal/11118
+forumPost: https://governance.aave.com/t/updated-proposal-aave-grants-dao-renewal/11289
```

Comment:
sorry for delayed review
this will just be ignored by all the aave governance uis i'm aware of, better just add a `references` section in the text and add all the links there (as discussions only supports a single link)